src/main/java/gregtech/common/command/CommandRecipeCheck.java

```diff
@@ -55,12 +56,25 @@ public void execute(@Nonnull MinecraftServer server, @Nonnull ICommandSender sen
         sender.sendMessage(new TextComponentTranslation("gregtech.command.recipecheck.begin"));
 
         Object2ObjectOpenHashMap<RecipeMap<?>, Object2ObjectOpenHashMap<Recipe, Set<Recipe>>> mismatchedRecipes = new Object2ObjectOpenHashMap<>();
+        Object2ObjectOpenHashMap<RecipeMap<?>, Set<Recipe>> emptyInputRecipes = new Object2ObjectOpenHashMap<>();
+        Set<Integer> emptyOreDicts = new ObjectOpenHashSet<>();
```

Comment:
```suggestion
        IntSet emptyOreDicts = new IntOpenHashSet();
```
The most minor review suggestion I've ever made